flake.nix

```
@@ -71,6 +71,42 @@
 
           pkgs = import nixpkgs { inherit system; };
 
+          # Download statically linked vmtest binary
+          arch = pkgs.lib.strings.removeSuffix "-linux" system;
+          vmtestVersion = "0.18.0";
+          vmtest = pkgs.stdenv.mkDerivation {
+            name = "vmtest";
+            version = vmtestVersion;
+            src = builtins.fetchurl {
+              url = "https://github.com/danobi/vmtest/releases/download/v${vmtestVersion}/vmtest-${arch}";
+              sha256 = "sha256:1wv49fq7n820jj7zyvbvrrzg2vwvyy8kb3gfw1lg55rzfqzhl9v3";
+            };
+            # Remove all other phases b/c we already have a prebuilt binary
+            phases = [ "installPhase" ];
+            installPhase = ''
+              install -m755 -D $src $out/bin/vmtest
+            '';
+          };
+
+          # Define lambda that returns a derivation for a kernel given kernel version and SHA as input
+          mkKernel = kernelVersion: sha256:
+            with pkgs;
+            pkgs.stdenv.mkDerivation rec {
```

Nit: don't need `pkgs.` with the `with pkgs;` above
```suggestion
            stdenv.mkDerivation rec {
```

---

Lemme drop the `with pkgs`. In #3751 we are moving to be more explicit. Would be good to keep it all consistent.